src/drivers/launch/encoder_odometry_node.launch

```diff
@@ -0,0 +1,33 @@
+
+<launch>
+
+    <!-- Start the Encoder to Odometry Node -->
+    <node name="encoder_odometry_node" pkg="drivers" type="encoder_odometry_node" output="screen" >
+        <!-- 
+            Encoder Ticks should be published as a JointState massage,
+            this defines what names in the JointState correspond to the left 
+            and right encoders
+        -->
+        <rosparam param="left_encoder_joint_name"> "left_encoder" </rosparam>
+        <rosparam param="right_encoder_joint_name"> "right_encoder" </rosparam>
+
+        <!-- 
+            Number of times per second to generate a new Odometry message.
+            Higher values are not neccesarily better, as enough time should 
+            be allowed between Odometry estimates so that we are estimating
+            a new state based on several encoder readings
+        -->
+        <rosparam param="odom_msg_refresh_rate"> 10 </rosparam>
+
+        <!-- The frame the Odometry messages will be generated in -->
+        <rosparam param="odom_frame"> "encoder" </rosparam>
+
+        <!-- 
+            Load the physical parameters that define the physical robot
+            see examples in the `config` folder.
+        -->
+        <rosparam command="load" 
+            file="$(find drivers)/launch/config/test_cart.yaml" />
```

Comment:
Is there a way for the config files to be loaded dynamically instead of having to change the launch file? If not, could we change the launch file to match the config file?

Reply:
Hmm, I don't think there is a way to load them dynamically. I definitely agree that this is a less then ideal solution though. What do you mean by "change the launch file"? Change the name to match? I'm a bit against changing the name, if only because it means we'll need to duplicate this launch file for every robot. We could add some higher level launch files that set the config file and then run most of the stuff in this launch file (and then have one of _those_ for each robot), but that might increase launch file structure complexity as well. @ValRat - could I get your thoughts on this as well?

Reply:
I mean, we would have to edit this launch file in any case where we want to use a different config yaml file. Is there maybe someway of doing it using the roslaunch command by feeding it a second paramater?

Reply:
So quick test shows that rosparam loaded variables are static until roscore dies. Maybe we can have people load the config files themselves? I know its one extra step, but imo it might be easier. Also loading other config files will just overwrite current rosparams anyways.

Reply:
Hmm, I'd prefer to automate as much as possible (the end goal being a single command to run as much of the system as possible), as I think it will really reduce simple avoidable errors, but I'll let @ValRat take the final call on this one.